index-flex1.html

```diff
@@ -56,16 +59,21 @@ <h2>The Past</h2>
             <h2 class="for-spacing">Why Did It Have To Be Boxes...</h2>
 
             <div class="boxes">
+            <div class="boxes1-5">
```

Comment:
I would leave the boxes all in one larger div as it was given. Work with the code that you had already been given. So start by display flexing the boxes class and go from there. What does that do? Then start playing with the flex-wrap. Figure out one puzzle piece at a time. You'll get it!